### ISSUE TITLE
Add a CI builder for Windows

### ISSUE THREAD

**Issue by zimmski** [sergi/go-diff#66]
I do not know of any free Windows CIs. So this is a very open question for me.

**Comment by maksimov** [comment]
Have you looked at https://www.appveyor.com/?

**Comment by zimmski** [comment]
No, but it looks promising. I am a little lost when it comes to Windows but IMHO go-diff must only compile and pass the tests on other OSs. I did a small search and these are maybe good resources to get started:
- https://blog.markvincze.com/setting-up-an-appveyor-pipeline-for-golang/
- https://github.com/joefitzgerald/go-plus/blob/master/appveyor.yml
- https://github.com/oschwald/maxminddb-golang/blob/master/appveyor.yml

**Comment by silvergasp** [comment]
I have implemented a config file for appveyor, you will just need to open an account with appveyor and it should find the file and start a build. Currently builds and tests are working however, lint is not working as it relies on a *nix shell. See Pull request [here](https://github.com/sergi/go-diff/pull/74)

**Comment by zimmski** [comment]
I think I do not have enough rights for this. Will have to ask @sergi
